taskExtraction/.env

```diff
@@ -0,0 +1 @@
+COHERE_API_KEY=zvkP21Kokbk0sIN2wBoouOA1L3G7M34QtX4pLaDo
```

Comment:
This should not be uploaded. 
I will add it to the gitignore. For now, we can each download it and use it locally.